src/css/custom.css

```
@@ -17,11 +30,15 @@
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
 }
 
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Atyp', fallbackFont, sans-serif; /* 'fallbackFont' is the name of a fallback font in case 'YourCustomFontName' doesn't load. */
```

This is missing the fallback font. Recommended to use system fonts, for example:

```suggestion
  font-family: 'Atyp', -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```